lib/gooddata/models/project.rb

```diff
@@ -272,6 +273,21 @@ def update_dataset_mapping(model_mapping_json, to_project)
         }
       end
 
+      def get_ldm_layout(from_project)
+        GoodData::LdmLayout.get(:client => from_project.client, :project => from_project)
+      end
+
+      def save_ldm_layout(ldm_layout_json, to_project)
+        ldm_layout = GoodData::LdmLayout.new(ldm_layout_json)
+        res = ldm_layout.save(:client => to_project.client, :project => to_project)
+        status = res&.dig('ldmLayout', 'layout').nil? ? "Failed" : "OK"
```

Comment:
In case network hiccup or invalid ldm_layout_json data. How about the result in this cases. Could you please help check them ?
Thank you.

Reply:
Currently,  when getting errors from the pipeline the pipeline will be broken. It's not the expectation, so I will fix to ignore the errors from sync_ldm_layout and the pipeline will continue.

Reply:
That is ok. When getting errors then please also write to log (warning log) the projectId of the error projects.
Thank you.

Reply:
Hi a @hung-nguyen-hoang,
I have updated to ignore errors from sync_ldm_layout and logging for this case.
Could you re-check? 
Thanks. 

![Screen Shot 2022-10-25 at 16 38 09](https://user-images.githubusercontent.com/63232274/197740136-1b294096-f21d-445e-aa81-cf70d420364f.png)

Reply:
Hi @khanhhuynh-gooddata,
It looks good to me. 
Thank you.